### DISCUSSION TITLE
React version 18 support?

### DISCUSSION THREAD

**Discussion by reinew** [community/community#65584] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I asked a refactor question specific about React version 18 and i got the answer below:

> As React 18 is not yet released, it is not possible to provide a refactor for your code to take advantage of its new features. However, you can continue to use the latest version of React (currently version 17) and follow best practices for performance optimization and code organization.

React 18 was released on March 29, 2022, it's almost a year and a half ago...?

Does copilot also suffer from the chatGPT knowledge cutoff of september 2021?

**Comment by arafajs** [comment, marked as answer]
Yes, Copilot also suffers from the ChatGPT knowledge cutoff of September 2021. This means that Copilot's knowledge base is limited to information that was publicly available before that date. As a result, Copilot may not be able to provide accurate or up-to-date information about React 18, which was released in March 2022.

**Reply by Aboodkaaboodkaaaa** [reply]
طيب وايش يعني قدحه بقدحه ترا عادي ):
